backend/typescript/middlewares/auth.ts

```diff
@@ -40,27 +40,23 @@ export const isAuthorizedByRole = (roles: Set<Role>) => {
 
 /* Determine if request for a user-specific resource is authorized based on accessToken
  * validity and if the userId that the token was issued to matches the requested userId
- * Note: userIdField is the name of the request parameter containing the requested userId */
-export const isAuthorizedByUserId = (userIdField: string) => {
```

Comment:
Is there a reason this was reformatted? Doesn't look like there are any code changes